testsuite/features/github_validation/init_clients/proxy_container.feature

```diff
@@ -0,0 +1,76 @@
+# Copyright (c) 2024 SUSE LLC
+# Licensed under the terms of the MIT license.
+#
+# The scenarios in this feature are skipped if:
+# * there is no proxy ($proxy is nil)
+#
+# Bootstrap the proxy as a Pod
+
+@proxy
+Feature: Setup containerized proxy
+  In order to use a containerized proxy with the server
+  As the system administrator
+  I want to register the containerized proxy on the server
+
+  Scenario: Check pod and container statuses
+    When I get the contents of the remote file "/tmp/podman-proxy.log"
+    Then it should contain a "uyuni-proxy-test-status: Running" text
+    And it should contain a "proxy-http-status: running" text
+    And it should contain a "proxy-ssh-status: running" text
+    And it should contain a "proxy-squid-status: running" text
+    And it should contain a "proxy-salt-broker-status: running" text
+    And it should contain a "proxy-tftpd-status: running" text
+    And it should contain a "uyuni-proxy-test-containers: 6" text
+
+  # Because of https://github.com/uyuni-project/uyuni/pull/8520 the rest is skipped.
+  @skip
+  Scenario: Log in as admin user
+    Given I am authorized for the "Admin" section
+
+  @skip
+  Scenario: Create an activation key for the Proxy
+    When I follow the left menu "Systems > Activation Keys"
+    And I follow "Create Key"
+    And I wait until I do not see "Loading..." text
+    And I enter "Proxy Key x86_64" as "description"
+    And I enter "PROXY-KEY-x86_64" as "key"
+    And I click on "Create Activation Key"
+    Then I should see a "Activation key Proxy Key x86_64 has been created" text
+
+  @skip
+  Scenario: Bootstrap the proxy host as a salt minion
+    When I follow the left menu "Systems > Bootstrapping"
+    Then I should see a "Bootstrap Minions" text
+    When I enter the hostname of "uyuni-proxy-test" as "hostname"
```

Comment:
I wonder if we could simplify this name, not sure how many files are affected by this, but as hostname, it would be simpler to call it "proxy".

Reply:
the server is called uyuni-server-test, so I guess this is why. I would prefer to have a similar name for the proxy , as uyuni-proxy-test.

Reply:
Yes, exactly. I think I will leave it like this and eventually we will rename both server and proxy one day.